src/main/java/net/glowstone/io/anvil/AnvilChunkIoService.java

```diff
@@ -150,7 +150,15 @@ public boolean read(GlowChunk chunk) throws IOException {
                 int tileY = tileTick.getInt("y");
                 int tileZ = tileTick.getInt("z");
                 String id = tileTick.getString("i");
-                Material material = Material.getMaterial(id);
+                if (id.startsWith("minecraft:")) {
```

Comment:
wrap the whole thing in this if statement, and else, say there was an unsupported a tile tick from a mod